types/array.prototype.find/auto.d.ts

```diff
@@ -0,0 +1,4 @@
+interface ReadonlyArray<T> {
+    find<S extends T>(predicate: (value: T, index: number, obj: T[]) => value is S, thisArg?: any): S | undefined;
+    find(predicate: (value: T, index: number, obj: T[]) => unknown, thisArg?: any): T | undefined;
+}
```

Comment:
FYI, I don't think this actually adds the types to the regular `Array` interface; it needs to be added separately

Reply:
sure, i can add both.